fastlane_core/lib/fastlane_core/ui/github_issue_inspector_reporter.rb

```diff
@@ -17,6 +17,7 @@ def inspector_successfully_recieved_report(report, inspector)
       report.issues[0..(NUMBER_OF_ISSUES_INLINE - 1)].each { |issue| print_issue_full(issue) }
 
       if report.issues.count > NUMBER_OF_ISSUES_INLINE
+        report.url.sub! '\'', '%27'
```

Comment:
Please surround method calls with `(` and `)`

Reply:
done and pushed.